sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -106,13 +106,13 @@ Below is a list of compile-time-constant properties which `pipe` supports.
 namespace sycl::ext::intel::experimental {
```

Comment:
I think at one point we discussed duplicating the properties both at this namespace and also in a lower avalon-st namespace to make autocomplete simpler for IDE users.

Reply:
Is there a compact or succinct way of specifying/writing this?

Reply:
Namespace aliases are the way to do it.  E.g.:

```
namespace a{
    namespace b {
        namespace c {
            struct foo {};
        }
    }
    using foo = b::c::foo;
}

a::b::c::foo f1;
a::foo f2;
```